### PR TITLE
Document ``php-ext-install``

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: config.rst [hunk 1]
```diff
@@ -473,8 +475,8 @@ tools:
 - `Symfony CLI <https://symfony.com/download>`_
 - `Composer <https://getcomposer.org/download/>`_
 
-Additionally, it creates four helpers: symfony-build_, symfony-deploy_,
-symfony-database-migrate_ and yarn-install_.
+Additionally, it creates five helpers: symfony-build_, symfony-deploy_,
```
Comment (fabpot) [inline config.rst:478]: it creates some helpers?

File: config.rst [hunk 2]
```diff
@@ -473,8 +475,8 @@ tools:
...
+symfony-database-migrate_, php-ext-install_ and yarn-install_.
```
Comment (fabpot) [inline config.rst:479]: , and

File: _includes/php-redis-warning.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
 Starting from PHP 7.4, the ``redis`` PECL extension is not longer included
-because this library had several API breaks. Instead, we recommend the use
-of the `predis package <https://packagist.org/packages/predis/predis/>`_
-which works across all supported PHP versions.
+because this library had several API breaks. Instead, we recommend to use
+our :ref:`php-ext-install <php-ext-install>` wrapper to pin-point the exact
```
Comment (fabpot) [inline _includes/php-redis-warning.rst:3]: the xxx script to install the exact...

File: config.rst [hunk 4]
```diff
@@ -527,6 +529,50 @@ symfony-deploy_ will make use of it. You can use this script at any moment if
...
+
+**php-ext-install** is a script that you can use to compile and install PHP
+extensions not provided :ref:`out of the box <php-extensions-list>` by
+SymfonyCloud. It is written specifically for SymfonyCloud to ensure fast and
+reliable setup during the :ref:`build step <build-hook>`. It currently supports
+three sources to fetch the sources:
+
+* From PECL: ``php-ext-install redis 5.2.2``
+
+* From an URL: ``php-ext-install redis https://github.com/phpredis/phpredis/archive/5.2.2.tar.gz``
```
Comment (fabpot) [inline config.rst:545]: From a URL

File: config.rst [hunk 5]
```diff
@@ -527,6 +529,50 @@ symfony-deploy_ will make use of it. You can use this script at any moment if
...
+
+* Cloning a Git repository: ``php-ext-install redis https://github.com/phpredis/phpredis.git 5.2.2``
```
Comment (fabpot) [inline config.rst:547]: From a Git repository

File: config.rst [hunk 6]
```diff
@@ -527,6 +529,50 @@ symfony-deploy_ will make use of it. You can use this script at any moment if
...
+
+            curl -s https://get.symfony.com/cloud/configurator | (>&2 bash)
+            (>&2
+                php-ext-install redis 5.2.2
+                symfony-build
+            )
+
+.. note::
+
+   Source code is cached between two builds and compilation is skipped if has
```
Comment (fabpot) [inline config.rst:567]: between builds

Comment (fabpot) [reply config.rst:567]: if it has

File: config.rst (hunk 1 above)
Comment (OskarStark) [reply config.rst:478]: It creates the following helpers....

Removing them would be bad when someone searchs for this.

Comment (tucksaun) [reply config.rst]: I think Fabien only wanted me to replace the number (to avoid updating it each time).
At least, this is what I did 😃

Comment (fabpot) [reply config.rst]: Yep :)

Comment (tucksaun) [reply config.rst]: 👍

**Comment by tucksaun** [comment]
`php-ext-install` is now available

File: config.rst [hunk 7]
```diff
@@ -527,6 +529,50 @@ symfony-deploy_ will make use of it. You can use this script at any moment if
...
+.. _php-ext-install:
+
+php-ext-install
+...............
+
+**php-ext-install** is a script that you can use to compile and install PHP
+extensions not provided :ref:`out of the box <php-extensions-list>` by
+SymfonyCloud. It is written specifically for SymfonyCloud to ensure fast and
+reliable setup during the :ref:`build step <build-hook>`. It currently supports
+three sources to fetch the sources:
```
Comment (fabpot) [inline config.rst:541]: to fetch the sources from:

**Pull request merged**